web-common/src/features/dashboards/tab-bar/TabBar.svelte

```diff
@@ -47,15 +47,17 @@
       selectedTab.label === "Pivot",
     );
 
-    behaviourEvent.fireNavigationEvent(
-      $metricsViewName,
-      BehaviourEventMedium.Tab,
-      MetricsEventSpace.Workspace,
-      MetricsEventScreenName.Dashboard,
-      selectedTab.label === "Pivot"
-        ? MetricsEventScreenName.Pivot
-        : MetricsEventScreenName.Explore,
-    );
+    if (behaviourEvent) {
```

Comment:
@AdityaHegde why do you think this is necessary? I see `web-admin`'s `initCloudMetrics()` function creates the `BehaviourEventFactory`